### ISSUE TITLE
Memory leak in backlight driver

### ISSUE THREAD

**Issue by selairi** [lxqt/liblxqt#250]
Sorry, I din not realize that I was in lxqt repository and I pushed this:
`[https://github.com/lxqt/liblxqt/commit/00a39490ca4befebabed364c5cfd76b89c8886fd](https://github.com/lxqt/liblxqt/commit/00a39490ca4befebabed364c5cfd76b89c8886fd)`

Backlight doesn't delete the backlight driver after use and it builds a new one.

##### Expected Behavior
Change the backlight.
After 60 seconds backlight driver is deleted.

##### Current Behavior
Backlight driver doesn't be deleted and new one is build.

##### Possible Solution
The solution is close the stream to backlight driver.
Sorry, I din not realize that I was in lxqt repo and I pushed the solution:
`[https://github.com/lxqt/liblxqt/commit/00a39490ca4befebabed364c5cfd76b89c8886fd](https://github.com/lxqt/liblxqt/commit/00a39490ca4befebabed364c5cfd76b89c8886fd)`
I was thinking that I was pushing to my own repo.

##### Steps to Reproduce (for bugs)
1. Open qps to check backlight driver
2. Change the backlight (lxqt-config-brightness) for several minutes.
3. Check qps and there are lots of backlight drivers

##### System Information
* Distribution & Version: 
* Kernel: 
* Qt Version: 
* libqtxdg Version: 0.14
* lxqt-build-tools Version: 
* Package version:

**Comment by tsujan** [comment]
@selairi 
https://github.com/lxqt/liblxqt/commit/00a39490ca4befebabed364c5cfd76b89c8886fd looks good to me, although I haven't had time to test it (I think you did).

**Comment by selairi** [comment]
@tsujan Yes, I test it. If you create a new LXQt::Backlight object and then you change the backlight, LXQt::Backlight will run a new backlight driver and it will kill it after 60 seconds.

I found the bug when I was testing the tool to change the backlight when computer is idle. For testing, I selected 5 seconds of idleness and when I checked the PID of lxqt-powermanager to kill it and run a new version, I found lots of backlight drivers running. A new one each 60 seconds. LXQt::Backlight was not killing the backlight driver.

**Comment by tsujan** [comment]
@selairi 
Yes, I'd guessed that you found and fixed the issue while testing your solution for https://github.com/lxqt/lxqt-powermanagement/issues//188

**Comment by selairi** [comment]
> @selairi
> Yes, I'd guessed that you found and fixed the issue while testing your solution for https://github.com/lxqt/lxqt-powermanagement/issues//188

Yes, you are right. I found it while testing that tool.
